controls/organization.sp

```diff
@@ -0,0 +1,238 @@
+locals {
+  organization_best_practices_common_tags = merge(local.github_insights_common_tags, {
+    service = "GitHub/Organization"
+  })
+}
+
+benchmark "org_best_practices" {
```

Comment:
```suggestion
benchmark "organization_best_practices" {
```

Since the dashboards use the full resource names, we can update these to be consistent as well